<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchPointRegistrationManipulatorControls</class>
 <widget class="QWidget" name="MatchPointRegistrationManipulatorControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>983</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupReg">
     <property name="title">
      <string>Source registration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioNewReg">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select this option to start with a new registration.&lt;/p&gt;&lt;p&gt;The registration will be pre initialized to map the center of the selected images.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>New registration (Center pre initialization)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSelectedReg">
        <property name="toolTip">
         <string>Use the selected registration as bsaeline. The manipulation will be &quot;on top&quot; of the given registration.</string>
        </property>
        <property name="text">
         <string>Selected registration</string>
        </property>
       </widget>
      </item>
      <item>
        <widget class="QmitkSingleNodeSelectionWidget" name="registrationNodeSelector" native="true">
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>40</height>
            </size>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="checkAutoSelect">
          <property name="toolTip">
            <string>If checked, the target and the moving image will be selected (if available) together with a selected pre registration.</string>
          </property>
          <property name="text">
            <string>Autoselect images by registration</string>
          </property>
          <property name="checked">
            <bool>true</bool>
          </property>
        </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Moving image:</string>
     </property>
    </widget>
   </item>
   <item>
     <widget class="QmitkSingleNodeSelectionWidget" name="movingNodeSelector" native="true">
       <property name="minimumSize">
         <size>
           <width>0</width>
           <height>40</height>
         </size>
       </property>
     </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Target image:</string>
     </property>
    </widget>
   </item>
   <item>
     <widget class="QmitkSingleNodeSelectionWidget" name="targetNodeSelector" native="true">
       <property name="minimumSize">
         <size>
           <width>0</width>
           <height>40</height>
         </size>
       </property>
     </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbStart">
     <property name="toolTip">
      <string>Starts the manipulation /manual registration.</string>
     </property>
     <property name="text">
      <string>Start manual registration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>New registration name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lbNewRegName">
     <property name="toolTip">
      <string>Name of the resulting manual registration</string>
     </property>
     <property name="text">
      <string>ManualRegistration</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkMapEntity">
     <property name="toolTip">
      <string>Check if a mapped moving image should be generated if the manual registration is confirmed.</string>
     </property>
     <property name="text">
      <string>Generate + store mapped result</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTransform">
      <attribute name="title">
       <string>Transformation settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QmitkRegistrationManipulationWidget" name="manipulationWidget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Center of rotation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboCenter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Allows to set the center of rotation.</string>
           </property>
           <item>
            <property name="text">
             <string>Moving image center</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>World origin</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current navigator position</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEval">
      <attribute name="title">
       <string>Evaluation settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QmitkRegEvalSettingsWidget" name="evalSettings" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="toolTip">
        <string>Cancel the manual registration and resets the state of the view.</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStore">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm the defined registration.&lt;/p&gt;&lt;p&gt;The registration will be stored (and, if activated, a mapped image will be generated).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Confirm + Store</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkRegEvalSettingsWidget</class>
   <extends>QWidget</extends>
   <header>QmitkRegEvalSettingsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkRegistrationManipulationWidget</class>
   <extends>QWidget</extends>
   <header>QmitkRegistrationManipulationWidget.h</header>
   <container>1</container>
  </customwidget>
   <customwidget>
     <class>QmitkSingleNodeSelectionWidget</class>
     <extends>QWidget</extends>
     <header location="global">QmitkSingleNodeSelectionWidget.h</header>
     <container>1</container>
   </customwidget>   
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
